toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -28,6 +28,8 @@ const (
 	toolchainRepoId        = "toolchain-repo"
 	cacheRepoID            = "upstream-cache-repo"
 	squashChrootRunErrors  = false
+	chrootRepoDir          = "/etc/yum.repos.d/"
+	chrootRepoFile         = "allrepos.repo"
```

Comment:
Could we move these back where they were, since they don't need to be globals anymore?